config/routes.rb

```diff
@@ -11,7 +11,11 @@
         resources :users, only: :update
         resources :topics, only: :index
         resources :targets, only: %i[create index destroy]
-        resources :conversations, only: :index
+        resources :conversations, only: :index do
+          namespace 'conversations', path: '' do
+            resources :messages, only: %i[create index]
```

Comment:
Is it necessary the namespace if you use the resource here?
If yes, can't you use `:conversations`? Is it necessary the `path: ''`?

Reply:
actually, it isn't necessary, who knows what was i trying to do 😆

Reply:
Actually, I do need the namespace, otherwise it looks for the controller on api/v1.
I put `path: ''` because otherwise
the route would be 
`/api/v1/conversations/:conversation_id/conversations/messages`

Reply:
Can't you use `resources :messages, module: :conversations`?